sbtc-core/src/signer/coordinator/mod.rs

```diff
@@ -0,0 +1,35 @@
+/// Module for FIRE signature generation
+pub mod fire;
+/// Module for ROAST signature generation
+pub mod roast;
+
+use std::collections::HashMap;
+
+use crate::SBTCResult;
+use bitcoin::{PublicKey, Transaction as BitcoinTransaction};
+use p256k1::ecdsa;
+use wsts::{bip340::SchnorrProof, common::Signature};
+
+#[derive(Default, Clone, Debug)]
+/// Signers' public keys required for weighted distributed signing
+pub struct PublicKeys {
+    /// Signer ids to public key mapping
+    pub signer_ids: HashMap<u32, ecdsa::PublicKey>,
+    /// Vote ids to public key mapping
+    pub vote_ids: HashMap<u32, ecdsa::PublicKey>,
+}
+
+/// TODO: Define the Message types for DKG round
+/// TODO: Define the Message types for Tx Signning Round
```

Comment:
Bonus points for linking issues in the todo comments

Reply:
Done